packages/cli/src/ui/hooks/useGeminiStream.ts

```diff
@@ -1010,10 +913,13 @@ export const useGeminiStream = (
     ],
   );
 
-  const pendingHistoryItems = [
-    pendingHistoryItemRef.current,
-    pendingToolCallGroupDisplay,
-  ].filter((i) => i !== undefined && i !== null);
+  const pendingHistoryItems = useMemo(
+    () =>
+      [pendingHistoryItemRef.current, pendingToolCallGroupDisplay].filter(
+        (i) => i !== undefined && i !== null,
+      ),
+    [pendingHistoryItemRef, pendingToolCallGroupDisplay],
```

Comment:
should this be
`pendingHistoryItemRef.current` 
in the useMemo
rather  than `pendingHistoryItemRef`

Reply:
i'm no longer using pendingHistoryItemRef